modules/snooker/src/test/ts/module/ephox/snooker/test/TestGenerator.ts

```diff
@@ -18,10 +19,16 @@ export default (): SimpleGenerators => {
     return r;
   };
 
-  const replace = (name: string) => {
-    const r = 'h(' + name + ')_' + replaceCounter;
-    replaceCounter++;
-    return r;
+  const replace = (elem: SugarElement) => {
+    if (elem?.dom) {
+      elem.dom.innerText = 'h(' + elem.dom.innerText + ')_' + replaceCounter;
```

Comment:
```suggestion
      TextContent.set(elem, `h(${ TextContent.get(elem) })_${replaceCounter}`);
```
While not 100% the same, as this uses `textContent` instead of `innerText` that shouldn't mater here from what I could tell.